app/jobs/split_pdf_job.rb

```diff
@@ -239,50 +239,69 @@ def save_pages(exam_template, partial_exams, filename=nil, split_pdf_log=nil)
         end
         repo.commit(txn)
 
-        # convert PDF to an image
+        next unless exam_template.automatic_parsing
         begin
+          # convert PDF to an image
           imglist = Magick::Image.from_blob(cover_pdf.to_pdf) do
             self.quality = 100
             self.density = '300'
           end
-        rescue Exception
```

Comment:
Don't change this. If there are other errors being raised by subsequent code, please discuss them with me.